README.md

```diff
@@ -44,6 +45,10 @@ barman_databases:                                 # Mandatory
     standby_hosts: "{{ groups['db-app1'][1:] }}"     # Optional (Automatically authorize SSH this servers list)
     extract_host_from_var: 'ec2_private_ip_address'  # Optional (host variable to extract from inventory hostvars)
 
+#Install a cron job and send logs to admin@domain.com
+barman_cron_enabled: True # default True
+barman_cron_notify: "admin@domain.com" # default ''
```

Comment:
could you please use `example.com` as example domain please?

Reply:
Updated